cfa_azure/helpers.py

```diff
@@ -927,9 +934,8 @@ def add_task_to_job(
         )
         batch_client.task.add(job_id=job_id, task=task)
         logger.debug(f"Task '{task_id}' added to job '{job_id}'.")
-        t = []
-        t.append(task_id)
-        return t
+        tasks.append(task_id)
+    return tasks

```

Comment:
this add_task() function shouldn't be changed. It's currently setup to return the string task_id name if it's just a single task that gets added and a list if it is multiple. Returning a string is necessary for the new automation functions